tpgtools/overrides/cloudbuildv2/beta/repository.yaml

```diff
@@ -3,3 +3,16 @@
   field: connection
   details:
     name: parent_connection
+# TODO: remove once DCL is updpated to not have these fields required.
+- type: CUSTOM_SCHEMA_VALUES
+  field: project
+  details:
+    required: false
+    optional: true
+    computed: true
+- type: CUSTOM_SCHEMA_VALUES
```

Comment:
Why do we need these? Can we just remove these from the DCL instead of having workarounds here?

Reply:
Yes, I'm removing them in cl/544465360. However, to not have to wait for the DCL release, I'm adding this override. I'll remove the override once the DCL fix is in place.